Apps/phdigitalshadows/digital_shadows.json

```diff
@@ -1039,35 +1171,25 @@
                     ]
                 },
                 "user_name": {
+                    "description": "User Name (use '*' for wildcard matching)",
                     "data_type": "string",
-                    "order": 4,
-                    "description": "User name (use '*' for wildcard matching)"
+                    "order": 4
                 },
                 "password": {
+                    "description": "Password (use '' for wildcard matching)",
```

Comment:
Empty '' ?

Reply:
Fixed

Reply:
Hi @somya-metron 
As the data type of this parameter has been changed to password, do we still need to add the wildcard matching related statement in the description?

Reply:
Yes, we will need the wildcard matching for the password field.

Reply:
okay, then we can consider this as fixed